scrowl-io/owl-ui
# [sc-299] - Create Card Component

### Short summary
Create a card component with multiple appearances. 

This component should contain a header, body and footer, and each one should render a wrapper for the content added by the developer when using the main Card component.

*The cards don't look nice because the designs were not provided yet for each variant so basic styles were added to get the component ready to receive the styles that will be provided.

**I also added a pretty standard CSS reset stylesheet to override styles added by the browser.

---

### Test steps
1. Run yarn start:storybook
2. Navigate through the different variants and ensure no errors pop up in the console.
